fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -278,6 +284,8 @@ public void execute() throws Exception {
         UUID uuid = context.getQueryId();
         context.setExecutionId(UUIDUtil.toTUniqueId(uuid));
         SessionVariable sessionVariableBackup = context.getSessionVariable();
+        LOG.info("begin execute query id = " + DebugUtil.printId(uuid));
```

Comment:
It's better to add to the LOG.debug()

Reply:
OK. but after second though,  I see there are begin/end time on profile, so there is no need to log it.